_layouts/en.html

```diff
@@ -18,6 +18,27 @@
             <h1 class="title">{{ page.title }}</h1>
           {% endunless %}
 
+          {% if page.deploy == 'v1' %}
+            <blockquote>
+              <p>
+                This page documents deployments using dpl v1 which currently is the default
+                version. The next major version dpl v2 will be released soon, and we recommend
+                starting to use it. Please see <a href="https://blog.travis-ci.com/2019-08-27-deployment-tooling-dpl-v2-preview-release">our blog post</a>
+                for details. Documentation for dpl v2 can be found <a href="/user/deployments">here</a>.
```

Comment:
```suggestion
                for details. <a href="/user/deployment">dpl v2 documentation can be found here</a>.
```

I believe in linking longer despcriptions than here, this helps with screen readers and building trust :) 

This also fixes the link from `/deployments` to `/deployment`